BootloaderCommonPkg/Include/Library/CryptoLib.h

```diff
@@ -14,8 +14,27 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #define RSA_KEY_VB_SIGNATURE     SIGNATURE_32 ('$', '_', 'V', 'B')
 #define RSA_KEY_IPP_SIGNATURE    SIGNATURE_32 ('$', 'I', 'P', 'P')
+#define PUBKEY_IDENTIFIER        SIGNATURE_32 ('P', 'U', 'B', 'K')
+#define SIGNATURE_IDENTIFIER     SIGNATURE_32 ('$', 'I', 'P', 'P')
+
 
 #define SIG_TYPE_RSA2048SHA256   0
+#define SIG_TYPE_RSA3072SHA384   1
```

Comment:
Where/How do we use this ?

Reply:
Current used in cryptolibs for RSA2048SHA256 check. It would be removed later.